docker/nl/docker-compose.yaml

```
@@ -20,20 +20,20 @@ version: "3.8"
 services:
   database:
     build:
-      context: source/database
+      context: ../common/database
       args:
-        - DATABASE_VERSION=${IMAGE_TAG}_${GITHASH}
-    image: ${REGISTRY_URL}emission-database:${IMAGE_TAG}
+        - DATABASE_VERSION=${AERIUS_IMAGE_TAG}_${AERIUS_GITHASH}
+    image: ${AERIUS_REGISTRY_URL}emission-database:${AERIUS_IMAGE_TAG}
     restart: unless-stopped
     volumes:
       - /etc/localtime:/etc/localtime:ro
 
   api:
-    build: source/rest-service
-    image: ${REGISTRY_URL}emission-rest-service:${IMAGE_TAG}
+    build: ../common/api
+    image: ${AERIUS_REGISTRY_URL}emission-rest-service:${AERIUS_IMAGE_TAG}
```

```suggestion
    image: ${AERIUS_REGISTRY_URL}api:${AERIUS_IMAGE_TAG}
```
maybe? might as well make it the same.

---

Registry URL should contain something related to the application then, but guess that's always the case anyway.

It is perhaps slightly weird that the registry URL contains a build-specific part when just looking at the name (at least I'd think that it would be a base URL to the registry, and that there is only 1 registry for the whole AERIUS landscape).

Adjusted it to the suggestion.